tests/src/test/scala/cats/tests/OptionTests.scala

```diff
@@ -76,4 +76,12 @@ class OptionTests extends CatsSuite {
       isEq.lhs should === (isEq.rhs)
     }
   }
+
+  // a test for OptionIdOps.some to return Some even if the argument is null
```

Comment:
This comment probably isn't really needed since the test name says basically the same thing.